bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```
@@ -263,8 +263,8 @@ regexp.invalid.unicode.general.category.value = invalid Unicode general category
 regexp.invalid.unicode.property.value = invalid Unicode property value ''{0}''
 regexp.empty.character.class.disallowed = empty character class disallowed
 regexp.invalid.hex.digit = invalid hexadecimal digit
-invalid.method.call = ''{0}'' method is called before module initialization
-invalid.function.invocation.call = function ''{0}'' is called before module initialization
+function.already.called = function ''{0}'' is already called
```

```suggestion
function.already.called = function ''{0}'' has already been called
```
Would this be better?